atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -129,16 +129,35 @@ public void create(byte[] tableMetadata) {
     }
 
     private void maybeAlterTableToHaveOverflowColumn() {
-        log.info("Potentially altering table {} to have overflow column.", LoggingArgs.tableRef(tableRef));
-        if (config.alterTablesOrMetadataToMatch().contains(tableRef)) {
-            log.info("Config contains table {}, checking if we can alter table.", LoggingArgs.tableRef(tableRef));
+        if (config.alterTablesOrMetadataToMatch().isEmpty()) {
+            return; // Short circuit
+        }
+        String shortTableName;
+        try {
+            shortTableName = oracleTableNameGetter.getInternalShortTableName(conns, tableRef);
+        } catch (TableMappingNotFoundException e) {
+            throw Throwables.rewrapAndThrowUncheckedException(
+                    "Unable to test whether table must be altered to have overflow column due to a table mapping"
+                            + " error.",
+                    e);
+        }
+
+        log.info(
+                "Potentially altering table {} to have overflow column.",
+                LoggingArgs.tableRef(tableRef),
+                UnsafeArg.of("shortTableName", shortTableName));
```

Comment:
Normally the `{}` needs to match the number of arguments being passed I think

Reply:
fixed in all the places I spotted that I changed